### PR TITLE
chore: add golden tests (#75)

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

**Comment by Vlix** [comment]
@blackheaven
If you could rebase this on `master`, that'd fix the CI again :)

**Comment by blackheaven** [comment]
Done, thanks.

**Review by Vlix — CHANGES_REQUESTED** [review]
Looks good overall, just a few things that look irregular

File: password/test/golden/Scrypt_outputLength16.golden
[the remark this answers is not in the document]
Comment (blackheaven) [reply password/test/golden/Scrypt_outputLength16.golden:1]: I didn't change the right parameters

**Review by Vlix — APPROVED** [review]

**Pull request merged**